test-scripts/coverage.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Do we want to dockerize this/other similar components going forward?

Reply:
Maybe, I'm holding off where I can adding docker stuff. I don't know that this makes sense to be docker since it's just running the tests, which we don't run in docker currently.